trans_first_transaction_express.resolved.rb

```diff
@@ -0,0 +1,941 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class TransFirstTransactionExpressGateway < Gateway
```

Comment:
This is the "conflict resolved" file. 
Use it as a reference and to look up any comments i left.

Before merging, remove this file